server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMailboxesMethodTest.java

```diff
@@ -69,6 +69,8 @@
     private static final String NAME = "[0][0]";
     private static final String ARGUMENTS = "[0][1]";
     private static final String FIRST_MAILBOX = ARGUMENTS + ".list[0]";
+    public static final String MAILBOX_NAME = FIRST_MAILBOX + ".name";
+    public static final String SHARED_WITH = FIRST_MAILBOX + ".sharedWith";
```

Comment:
I don't think these constants are useful. I think matthieu was more talking about replacing these ugly ImmutableList.of("e", "r", "s", "t", "w")

Reply:
Is that your comment @mbaechler  ?

Because as such I struggle to find  something more readable than the **ugly** ImmutableList.of(...). At least it clearly displays the rights, and do not requires inspections to know what it means.

If you prefer a constant named RIGHTS_ERSTW with the list extracted, well, I don't.

Reply:
Ok, let me write some code snippet, I suggest : 
```
ImmutableList.of(Expunge, Read, blabla)
```

Reply:
Ok with Expunge = "e" as a constant, right?

Reply:
just need to revert this commit, and I think we are good

Reply:
done + squashing